engine/class_modules/paladin/sc_paladin.cpp

```diff
@@ -950,6 +965,13 @@ struct vanquishers_hammer_t : public holy_power_consumer_t<paladin_melee_attack_
     holy_power_consumer_t::impact( s );
 
     p() -> buffs.vanquishers_hammer -> trigger();
+
+    if ( p() -> conduit.righteous_might -> ok() )
+    {
+      r_m_heal -> base_dd_min = r_m_heal -> base_dd_max =
+        s -> result_amount * p() -> conduit.righteous_might.percent();
+      r_m_heal -> execute();
+    }
```

Comment:
Isn't the conduit value only for the damage increase on VH, with the conversion rate to healing being a static 100%?